roles/ohpc_matlab/tasks/main.yaml

```diff
@@ -0,0 +1,12 @@
+- name: install the latest version of libXtst
+  yum:
+    name: libXt
+    state: latest
+
+- name: Download matlab
+  get_url:
+    url: "{{ matlab_download_url }}"
+    dest: "{{ matlab_destination  }}"
+
+- name: Un-tar downloaded MATLAB 
+  command: tar -xvf {{ matlab_destination  }} -C "{{ matlab_clustershare }}"
```

Comment:
I'm getting an error on this task because the "-C" tries to change to a directory that doesn't exist.  You may need a task that creates the path first.

Also remove the "v" from the tar command.  There is no need to enumerate the file listing during extraction.

I also got a warning to consider using the unarchive module.  Could this be a better solution here?

Reply:
I believe the -C in tar creates the directory if not found (http://man.openbsd.org/tar)
Removing v in the tar command

Reply:
That man page doesn't explicitly say it will be created, just that it is extract to that directory.  Also, we are on linux not openbsd so take care to ensure feature parity.  BSD and FSF versions of core tools sometimes have slightly different behavior in these edge cases.

Reply:
Also you might try the unarchive module and see if that works better than an explicit tar command